packages/expo-media-library/CHANGELOG.md

```diff
@@ -4,6 +4,8 @@
 
 ### 🛠 Breaking changes
 
+- Add camera roll permissions on Android. ([#9231](https://github.com/expo/expo/pull/9231) by [@bycedric](https://github.com/bycedric))
```

Comment:
I think we should get rid of "camera roll" name, it might be kinda confusing with `react-native-camera-roll`. Also, can you update the entry to make it more descriptive for users? Like:
```suggestion
- Added external storage permissions declarations to `AndroidManifest.xml` on Android. ([#9231](https://github.com/expo/expo/pull/9231) by [@bycedric](https://github.com/bycedric))
```